docs/dependencies.md

```diff
@@ -0,0 +1,72 @@
+
+# How to deal with dependencies
+
+For work with dependencies in Leapp repositories, you need to understand the
+dependencies types and why Leapp has a special way to deal with dependencies.
+
+Read this
+[Leapp documentation](https://github.com/oamg/leapp/blob/master/docs/source/dependencies.md)
+to understand it.
+
+## What to do when dependencies needs to be changed
+
+You can add dependencies for all repositories or only for one repository
```

Comment:
Nope. There is nothing like that. Both things you described are now basically for all repositories. When we will add additional repository, we will have to rethink completely how to deal with packaging. Probably packaging will have to be hidden under repositories itself. The thing is whether you are resolving dependencies for target system or src system (system where you install  the leapp).